stake-pool/cli/src/main.rs

```diff
@@ -1581,8 +1826,8 @@ fn main() {
                 .required(true)
             )
         )
-        .subcommand(SubCommand::with_name("deposit")
-            .about("Add stake account to the stake pool")
+        .subcommand(SubCommand::with_name("deposit-stake")
+            .about("Deposit fully-activated stake account into the stake pool in exchange for pool tokens")
```

Comment:
This is a nice clarification. We've had people confused about using the term `fully-activated` in the past, so just a slight simplification:
```suggestion
            .about("Deposit active stake account into the stake pool in exchange for pool tokens")
```